doc/release-notes/0.30/0.30.md

```diff
@@ -78,6 +85,15 @@ The v0.30.0 release contains the following known issues and limitations:
 <th valign="bottom">Impact</th>
 <th valign="bottom">Workaround</th>
 </tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/14192">#14192</th>
```

Comment:
Are the closing `</th>` tags correct here?  I see each line opens with a `<td` tag but closes with a `</th>` which seems mismatched.

Reply:
Thanks, fixed to use closing `</td>`.